src/main/cli/cli.c

```diff
@@ -1316,14 +1316,16 @@ static void cliSerial(const char *cmdName, char *cmdline)
     uint8_t validArgumentCount = 0;
 
     const char *ptr = cmdline;
-
-    int val = atoi(ptr++);
+    char *tok = cmdline;
+    int val = findSerialPortByName(strsep(&tok, " "), strcasecmp);
     serialPortConfig_t *currentConfig = serialFindPortConfigurationMutable(val);
```

Comment:
This may return non-null if SERIAL_PORT_NONE is passed as argument and some config has NONE in id